File: src/main/java/org/cloudfoundry/promregator/auth/NullEnricher.java
package org.cloudfoundry.promregator.auth;

import org.apache.hc.client5.http.classic.methods.HttpGet;

/**
 * The NullEnricher is an AuthenticationEnricher, which does not enrich 
 * the HTTP request. It can be used in case an AuthenticationEnricher is required,
 * but no operation shall be performed
 *
 */
public class NullEnricher implements AuthenticationEnricher {

	@Override
	public void enrichWithAuthentication(HttpGet httpget) {
		// left blank intentionally
	}

}

File: src/main/java/org/cloudfoundry/promregator/lite/auth/NullEnricher.kt
package org.cloudfoundry.promregator.lite.auth

import org.springframework.http.HttpHeaders

/**
 * The NullEnricher is an AuthenticationEnricher, which does not enrich
 * the HTTP request. It can be used in case an AuthenticationEnricher is required,
 * but no operation shall be performed
 *
 */
class NullEnricher : AuthenticationEnricher {
    override fun enrichWithAuthentication(headers: HttpHeaders) {
        // left blank intentionally
    }
}
